_posts/2025-03-12-orange-telco-core-network-api-management-with-quarkus.adoc

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: 'How Orange leverages Quarkus for seamless access to Telco network capabilities'
+date: 2025-03-12
+tags: user-story
+synopsis: Learn how Orange successfully deployed APIs across various 4G/5G network cores.
+author: tsegismont
+thumbnailimage: /assets/images/posts/quarkus-user-stories/orange/orange-logo.png
+---
+
+:imagesdir: /assets/images/posts/quarkus-user-stories/orange
+
+= How Orange leverages Quarkus for seamless access to Telco network capabilities
+
+image::orange-logo.png[alt=Orange,align=center,width=256]
+
+== Introduction
+
+As a global telecommunications leader, https://www.orange.com[Orange] has always been at the forefront of innovation.
+Along with Deutsche Telekom, Telefónica, and Vodafone, Orange co-founded the https://camaraproject.org[Camara Project], an initiative aimed at simplifying the consumption of 5G APIs for third-party application developers.
+
+To achieve this goal, Orange needed a flexible and lightweight framework capable of handling constrained API exposure while ensuring compatibility with existing network core systems.
+
+After a rigorous evaluation of multiple frameworks and toolkits — including Quarkus, Ktor, Micronaut, and Vert.x — Orange selected Quarkus as the ideal solution.
+
+https://www.linkedin.com/in/patriceconil/[Patrice Conil], software craftsman at Orange, told us why.
+
+== Why Quarkus? A Framework Evaluation
+
+The team at Orange embarked on a comparative study to test alternatives to Spring Boot in a well-defined API wrapper exposure context.
```

Comment:
It's unclear why they wanted a Spring Boot alternative?

Reply:
Because spring startup time can be a barrier to dynamic pod management